### PR TITLE
Add YCSB Benchmark

### PR DESCRIPTION
**Pull request by feeblefakie**

### CODE REVIEW

File: scalardb-test/src/main/java/kelpie/scalardb/ycsb/WorkloadF.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  @Override
+  public void executeEach() throws TransactionException {
+    DistributedTransaction transaction = manager.start();
+
+    try {
+      for (int i = 0; i < opsPerTx; ++i) {
+        int userId = ThreadLocalRandom.current().nextInt(recordCount);
+        transaction.get(prepareGet(userId));
+        // String payload = RandomStringUtils.randomAlphabetic(payloadSize);
```
Comment (feeblefakie) [inline scalardb-test/src/main/java/kelpie/scalardb/ycsb/WorkloadF.java]:
```suggestion
```

**Comment by brfrn169** [comment]
Thank you! It looks like this is very useful.

@feeblefakie I have a question. It looks like it just emulates the YCSB workloads, so we can use it when we don't want to use the YCSB tool but we want to use Kelpie for the YCSB workloads?

**Comment by feeblefakie** [comment]
@brfrn169 

> I have a question. It looks like it just emulates the YCSB workloads, so we can use it when we don't want to use the YCSB tool but we want to use Kelpie for the YCSB workloads?

Thank you!
I created them more like for database transactions since YCSB doesn't really handle transactions.
(So, I think many transaction papers that use YCSB workloads create their own benchmark on the basis of the specs.)

File: scalardb-test/src/main/java/kelpie/scalardb/ycsb/Common.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
+package kelpie.scalardb.ycsb;
+
+import com.scalar.db.api.Consistency;
+import com.scalar.db.api.Get;
+import com.scalar.db.api.Put;
+import com.scalar.db.io.Key;
+import com.scalar.kelpie.config.Config;
+import java.util.Random;
+
+public class Common {
```
Comment (brfrn169) [inline scalardb-test/src/main/java/kelpie/scalardb/ycsb/Common.java]: Maybe `YcsbCommon` is a better name as we already have `Common` in the `kelpie.scalardb` package?

**Review by brfrn169 — COMMENTED** [review]
Sorry for the late review.

I just left a minor comment. Please take a look when you have time!

File: scalardb-test/src/main/java/kelpie/scalardb/ycsb/Common.java (hunk 2 above)
Comment (feeblefakie) [reply scalardb-test/src/main/java/kelpie/scalardb/ycsb/Common.java]: Agreed. Fixed in 9727209. Thank you!

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**